src/pudl/constants.py

```diff
@@ -2577,6 +2578,8 @@
         'contract_type_code': pd.StringDtype(),
         'county': pd.StringDtype(),
         'county_id_fips': pd.StringDtype(),  # Must preserve leading zeroes
+        'critical_peak_pricing': pd.StringDtype(),  # Added by AES for DP table
+        'critical_peak_rebate': pd.StringDtype(),  # Added by AES for DP table
```

Comment:
Are all these new cols really `StringDtype()` or should they be `BooleanDtype()` (see below)

Reply:
Can they be `BooleanDtype()` if there are NA values embedded?

Reply:
Yes, if they're the **pandas** `BooleanDtype()` you're all good.  The native Python `bool` type won't work though.